chips/ibex-super-system/openocd.gdb

```diff
@@ -0,0 +1,10 @@
+target extended-remote :3333
+
+# print demangled symbols
+set confirm off
+set print asm-demangle on
+
+# set backtrace limit to not have infinite backtrace loops
+set backtrace limit 32
+
+monitor arm semihosting enable
```

Comment:
arm?

Reply:
This is how semihosting has to be enabled, presumably because it originated from ARM. There is no support for specifying `riscv` as far as I am aware.

Reply:
aah that is interesting. I didn't know that. Thanks!

Reply:
The semihosting feature is under `arm` for all platforms, but since our target is riscv, openocd should use the correct semihosting.

Reply:
> The semihosting feature is under arm for all platforms,

Is this within openocd? or humility?

Reply:
Openocd.  The `monitor` command tells gdb to pass the arguments to the underlying gdb server, in this case openocd.  Our other gdb servers (qemu) will just ignore this command.

Reply:
Ah okay. THat is weird. But okay.